view/frontend/web/js/fotorama-video-events-mixin.js

```diff
@@ -0,0 +1,60 @@
+define([
+    'jquery'
+], function ($) {
+    'use strict';
+
+    return function (widget) {
+        $.widget('mage.AddFotoramaVideoEvents', widget, {
+
+            _clickHandler: function (e) {
+                const blockVideoConsentConfig = window.cookiebotConfig && window.cookiebotConfig.blockVideosUntilConsent;
+
+                if (!Cookiebot?.consent?.marketing && blockVideoConsentConfig) {
+                    const videoElement = event.target.querySelector('.product-video');
+                    if (!Cookiebot?.consent?.marketing) {
+                        if (Cookiebot?.consent?.marketing) {
```

Comment:
This seems to be a duplicate condition...? It would be nice if we could use an early return here.